docs/docs-content/clusters/edge/third-party-integrations/tailscale.md

```diff
@@ -0,0 +1,249 @@
+
+- Kubernetes cluster pod CIDR
+- Kubernetes cluster service CIDR
+- Palette Edge Overlay network CIDR
+
+#### Debug Steps
+
+If you want to use parts of the 100.64.0.0/10 range for your Kubernetes clusters or your Palette Edge Overlay networks, you must limit the IP address range that your Tailnet uses to a fraction of the 100.64.0.0/10 range. Use the following steps to limit your Tailnet range:
+
+1. First, configure an IP Pool in Tailscale. We have found the following configuration works well to assign addresses in the new range to all nodes:
+
+```json
+	"nodeAttrs": [
+		{
+			"target": ["*"],
+			"ipPool": ["100.74.0.0/16"],
+		},
+	],
+```
+
+2. Next, in the OS pack of your cluster profile, add the following:
+
+```yaml
+stages:
+  initramfs:
+    - name: "Tailscale fix systemD unit service"
+      files:
+        - path: /etc/systemd/system/tailscale-iptables-fix.service
+          permissions: 0644
+          owner: 0
+          group: 0
+          content: |
+            [Unit]
+            Description=Tailscale iptables fix service
+            [Service]
+            ExecStart=/etc/palette/tailscale-iptables.sh
+            [Install]
+            WantedBy=multi-user.target
+    - name: "Tailscale fix systemD unit timer"
+      files:
+        - path: /etc/systemd/system/tailscale-iptables-fix.timer
+          permissions: 0644
+          owner: 0
+          group: 0
+          content: |
+            [Unit]
+            Description=Tailscale iptables fix schedule
+            [Timer]
+            OnBootSec=15
+            OnUnitActiveSec=15
+            [Install]
+            WantedBy=timers.target
+    - name: "Tailscale adjustment script"
+      files:
+        - path: /etc/palette/tailscale-iptables.sh
+          permissions: 0755
+          owner: 0
+          group: 0
+          content: |
+            #!/bin/sh
+            if iptables -L ts-input | grep DROP | grep 100.64.0.0/10; then
+              RULEFWD=$(iptables -L ts-forward --line-numbers | grep DROP | grep 100.64.0.0/10 | awk '{print $1}')
+              RULEINP=$(iptables -L ts-input --line-numbers | grep DROP | grep 100.64.0.0/10 | awk '{print $1}')
+              iptables -R ts-forward $RULEFWD -s 100.74.0.0/16 -o tailscale0 -j DROP
+              iptables -R ts-input $RULEINP -s 100.74.0.0/16 -o tailscale0 -j DROP
+            fi
+  network:
+    - name: "Reduce scope of traffic dropped by Tailscale to just the Tailscale ipPool"
+      commands:
+        - |
+          systemctl enable tailscale-iptables-fix.service
+          systemctl enable tailscale-iptables-fix.timer
+          systemctl start tailscale-iptables-fix.timer
+```
+
+This will ensure Tailscale does not drop traffic for IP ranges that it doesn't own. This is due to a known bug in Tailscale. Even though we restricted the IP Pool, Tailscale still puts in `iptables` rule on every node that drops unknown traffic from any address in the entire 100.64.0.0/10 range. 
```

Comment:
If we can link to the bug, that'd be awesome.

Reply:
Kevin said that he only managed to confirm it via email and this is not officially disclosed by tailscale